Dockerfile-3.5

```diff
@@ -8,7 +8,7 @@ RUN pip install -r requirements.txt -r requirements-test.txt
 
 COPY . /opt/project
 
-RUN pip install -e .
```

Comment:
@sevikkk Makes sense. I was just casting a wide net to get something up and confirm the failure and then fix. Will clean that up shortly.